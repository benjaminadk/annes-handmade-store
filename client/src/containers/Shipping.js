import React from 'react'
import { withStyles } from '@material-ui/core/styles'
import Typography from '@material-ui/core/Typography'

const styles = theme => ({
  root: {
    marginTop: '10vh',
    marginBottom: '10vh',
    paddingLeft: '5vw',
    paddingRight: '5vw',
    minHeight: '60vh'
  },
  title: {
    marginBottom: '5vh'
  },
  content: {
    width: '50%'
  }
})

const Shipping = ({ classes }) => (
  <div className={classes.root}>
    <Typography variant="display2" className={classes.title}>
      Shipping Policy
    </Typography>
    <Typography variant="body2" align="justify" className={classes.content}>
      We offer free shipping to all our customers. At this time we only ship
      within the United States. All orders are shipped within 3 business days
      pending credit card verification. All items are shipped via USPS and
      enclosed in padded packaging.
    </Typography>
  </div>
)

export default withStyles(styles)(Shipping)
