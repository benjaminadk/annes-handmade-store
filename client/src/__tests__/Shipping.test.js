import React from 'react'
import Shipping from '../containers/Shipping'
import renderer from 'react-test-renderer'

test('Shipping Page renders correctly', () => {
  const tree = renderer.create(<Shipping />).toJSON()
  expect(tree).toMatchSnapshot()
})
