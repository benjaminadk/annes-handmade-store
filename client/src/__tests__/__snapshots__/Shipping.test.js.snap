// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shipping Page renders correctly 1`] = `
<div
  className="Shipping-root-1"
>
  <h1
    className="MuiTypography-root-4 MuiTypography-display2-7 Shipping-title-2"
  >
    Shipping Policy
  </h1>
  <aside
    className="MuiTypography-root-4 MuiTypography-body2-12 MuiTypography-alignJustify-19 Shipping-content-3"
  >
    We offer free shipping to all our customers. At this time we only ship within the United States. All orders are shipped within 3 business days pending credit card verification. All items are shipped via USPS and enclosed in padded packaging.
  </aside>
</div>
`;
